packages/editor/src/plugins/text/utils/static-extract-text.ts

```diff
@@ -9,7 +13,19 @@ export function extractDescendant(node: Descendant): string {
   return node.text ? node.text.trim() : ''
 }
 
-export function extractStringFromTextDocument(document?: AnyEditorDocument) {
+export function extractStringFromTextDocument(
+  document?: AnyEditorDocument
+): string {
+  if (
+    document &&
+    isMultimediaDocument(document) &&
+    isRowsDocument(document.state.explanation)
+  ) {
+    return document.state.explanation.state
+      .map(extractStringFromTextDocument)
+      .join(' ')
+      .trim()
+  }
```

Comment:
Since this is a helper in a file inside the text plugin and is named `extractStringFromTextDocument` I'd prefer this code to be somewhere else.
Imho. this should be a generic helper in e.g. `/plugin/helpers/static-extract-text-from-document` or something like that.

Reply:
(or inside multimedia plugin)

Reply:
I have move the code to the one place it's used.